### DISCUSSION TITLE
Trying out the Heresy mod, can't enter cathedral?

### DISCUSSION THREAD

**Discussion by answerpoet** [flareteam/flare-game#956] [Q&A]
Playing on Win11 Pro.

I talked to everyone in the town and got the "clean up the basement" quest, but I can't seem to enter the rear door of the cathedral. 
Shouldn't I just have to walk onto the blue triangles at the bottom of the steps? TIA!

**Comment by dorkster** [comment]
You need to talk to the blacksmith, Greywold, first. He gives you your starting weapon and armor.

**Reply by answerpoet** [reply]
I talked to everyone, Greywold first. He pointed me to the church dude who gave me the basement quest. I walk around to the back entrance and nothing. Dunno.

**Reply by dorkster** [reply]
Greywold gets some new dialog *after* talking to Bishop Lucian (the church dude). He should have this dialog choice: **"Weapons on sale"**.

**Reply by answerpoet** [reply]
ahh.. thx
